### ISSUE TITLE
DataLoader worker (pid(s) 18056, 20540, 4512) exited unexpectedly

### ISSUE THREAD

**Issue by Aidenfaustine** [lstappen/MuSe2021#2]
I'm new to pytorch. when i run commands in Jupyternotebook （windows  systems） %run main.py --task wilder --emo_dim arousal --feature_set egemaps --normalize --norm_opts y --d_rnn 128 --rnn_n_layers 4 --epochs 100 --batch_size 1024 --n_seeds 10 --win_len 200 --hop_len 100 --use_gpu --cache --save --save_path preds
it reported error as follows:
DataLoader worker (pid(s) 18056, 20540, 4512) exited unexpectedly
Thanks
best wishes.

**Comment by lc0197** [comment]
Hi,
the code is not meant to be run in a notebook and not tested for that. Please try to run it as a command line script as suggested in the readme. Moreover, note that depending on feature set and batch size, the memory requirements can be too high for a standard computer to handle.

**Comment by Aidenfaustine** [comment]
Dear professor:

I'm new to pytorch, and don't have any background about cs and ee. If i want to run it, where should i =hange num_workers=0?
Thanks
